integrations/elasticsearch/pyproject.toml

```diff
@@ -21,7 +21,11 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: CPython",
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
-dependencies = ["haystack-ai", "elasticsearch>=8,<9"]
+dependencies = [
+                "haystack-ai>=2.11.0",
+                "elasticsearch>=8,<9",
+                "aiohttp" # for async support https://elasticsearch-py.readthedocs.io/en/latest/async.html#valueerror-when-initializing-asyncelasticsearch
```

Comment:
unrelated but see the failing tests https://github.com/deepset-ai/haystack-core-integrations/actions/runs/13787069988/job/38557478659 and the attached link to docs